js/packages/components/buttons/horizontal/TwoHorizontalButtonsSmallMargin.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactNode } from 'react'
+import { View } from 'react-native'
+
+import { useStyles } from '@berty/contexts/styles'
+
+interface TwoHorizontalButtonsSmallMarginProps {
+	children: ReactNode[]
+}
+
+export const TwoHorizontalButtonsSmallMargin: React.FC<TwoHorizontalButtonsSmallMarginProps> =
```

Comment:
Maybe just `TwoHorizontalButtons` for the naming ? Because we have just this component for the `horizontal` folder 👍 
(Same for the file and the interface naming)

Reply:
Sorry i didn't see that we have an other component in this folder, so the naming seems fine for me 👍